lib/oban/queue/producer.ex

```diff
@@ -221,7 +227,13 @@ defmodule Oban.Queue.Producer do
   end
 
   defp start_jobs(%State{} = state) do
-    {:ok, {meta, jobs}} = Engine.fetch_jobs(state.conf, state.meta, state.running)
+    tele_meta = tele_meta(state)
+
+    {meta, jobs} =
```

Comment:
This is already covered by the `dispatch` function, though I admit the `[:oban, :producer, _]` namespace isn't very helpful.

Reply:
Ah I see, so just `[:oban, :engine, :refresh]` then?

Reply:
Never mind, I misunderstood you, I'll move the Engine events to the engine module.